lib/util/versioning.js

```diff
@@ -55,6 +69,8 @@ function get_runtime_abi(runtime, target_version) {
     }
     if (runtime === 'node-webkit') {
         return get_node_webkit_abi(runtime, target_version || process.versions['node-webkit']);
+    } else if (runtime === 'electron') {
+        return get_electron_abi(runtime, target_version || process.versions.electron);
```

Comment:
So, the policy that https://github.com/electronjs/electron-rebuild uses is, assume your package is a sibling package of `electron-prebuilt`, then use https://github.com/electronjs/electron-rebuild/blob/master/src/main.js#L42 to get the version info. 

This isn't perfect, but it makes the Easy case Easy, and if you let devs also specify explicitly, you have an escape hatch